microsite/blog/2023-08-17-expedia-proof-of-value-metrics-2.mdx

```diff
@@ -0,0 +1,206 @@
+---
+# prettier-ignore
+title: "Expedia Group shares Backstage proof of value metrics 2.0"
+author: Guillermo Manzo, Expedia Group
+authorURL: https://www.linkedin.com/in/guillermomanzo
+---
+
+![backstage header](assets/2023-08-17/header.png)
+
+_Editor's Note: In this post, Expedia Group provides a follow-up to their initial Q4 2021 [post on how they measure Backstage proof of value ](https://backstage.spotify.com/blog/measuring-backstage-proof-of-value-at-expedia/).
+We're grateful to the Expedia Group team for sharing their processes, revised proof of value metrics, and adaptations to their approach over time in hopes of providing a successful framework for the rest of the Backstage community._
+
+[Expedia Group's](https://expediagroup.com/home/default.aspx) mission is to power global travel for everyone, everywhere.
```

Comment:
```suggestion
[Expedia Group's](https://expediagroup.com) mission is to power global travel for everyone, everywhere.
```

Feel free to ignore this if Expedia preferred the full URL - but these show the same content and my suggestion feels less likely to break in the future.